.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+common: &common
+  working_directory: ~/raster-tester
+  steps:
+    - checkout
+    - run: sudo apt-get -y --no-install-recommends --force-yes install gdal-bin libgdal-dev
```

Comment:
If we're using Rasterio wheels, we could get away with not installing GDAL, right?